versions/1/changelog.yml

```diff
@@ -79,4 +79,6 @@
   - description: Deprecate JSON-formatted pipeline test configs in favor of YAML-formatted ones
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/139
-
+  - description: Add rules package type with kibana/rules and kibana/timelines
```

Comment:
Are the two tied together? Otherwise I would split rules and timeline in two separate PR's.

Reply:
Done. Removed timelines from the scope of this PR